doc/modules/changes/20211221_haoyuanli

```diff
@@ -0,0 +1,5 @@
+New: I changed the variables of "min_visc" and "max_visc" in the material model to "minimum viscosity" and "maximum viscosity".
+Then I expand their implementation so that values from different phases could be imported.
+This is useful in assigning shear zone viscosity in a subduction model.
```

Comment:
The purpose of these notes is mostly to explain to users what changes in the behavior of the code, so anything that is only happening inside does not need to be mentioned (like renaming variables, unless you change them in the parameter file too). I would reformulate like this (properly broken into shorter lines):
```suggestion
New: The variables "minimum viscosity" and "maximum viscosity" in the visco plastic material model can now have different values for different phases. This is useful in assigning shear zone viscosity in a subduction model.
```